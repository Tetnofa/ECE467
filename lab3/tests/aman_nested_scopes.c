void main() {
    int x;
    bool y;
    {
        int x;  // Shadowing outer x
        x = 5;
        {
            bool x;  // Shadowing both outer x's
            x = true;
            {
                int y;  // Shadowing outer y
                y = 10;
                writeInt(y);
            }
            writeBool(x);
        }
        writeInt(x);
    }
    x = 1;  // Outer most x
    writeInt(x);
} 