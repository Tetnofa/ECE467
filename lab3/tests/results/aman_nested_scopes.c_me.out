sema: 13:16 : use of undefined identifier writeInt
sema: 15:12 : use of undefined identifier writeBool
sema: 17:8 : use of undefined identifier writeInt
sema: 20:4 : use of undefined identifier writeInt
